pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -1103,7 +1102,7 @@ class Timestamp(_Timestamp):
 
 
 # Add the min and max fields at the class level
-cdef int64_t _NS_UPPER_BOUND = INT64_MAX
```

Comment:
future TODO, we look up these limits all over the place. maybe make a ``limits.pyx`` somewhere